frontend/packages/dev-console/src/utils/add-resources-menu-utils.ts

```diff
@@ -78,6 +78,6 @@ export const createKebabAction: KebabFactory = (label, icon, importType, checkAc
     icon,
     path: getMenuPath(hasApplication, connectorSourceContext),
     href: getAddPageUrl(obj, importType, hasApplication, connectorSourceContext),
-    accessReview: checkAccess && asAccessReview(resourceModel, obj, 'create'),
+    accessReview: checkAccess ? asAccessReview(resourceModel, obj, 'create') : null,
```

Comment:
This fixes a warning that shows up in the console stating a boolean was being passed when an object is expected.

Reply:
https://github.com/openshift/console/pull/4070

Reply:
I've removed the change for easy rebasing..